policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/childless/max_amount.yaml

```
@@ -1,10 +1,13 @@
 description: Maryland provides up to this amount for its Earned Income Tax Credit to single childless filers.
 values:
   2019-01-01: 530
+  2023-01-01: 600
```

Per the legal code:
>(iii) For a taxable year beginning after December 31, 2019, but **before January 1, 2023,** the tax credit allowed under this paragraph may not exceed $530 for a taxable year.

And [this bill](https://mgaleg.maryland.gov/2023RS/fnotes/bil_0007/hb0547.pdf):
>Further, the $530 limit on the value of the State refundable credit for individuals without qualifying children is eliminated.

The comptroller report is just referencing the current federal EITC value for filers without qualifying children.

```suggestion
  2023-01-01: .inf
```

---

@MaxGhenis,  thanks for pointing this out.   Revisions have been made along the lines you suggested.

---

@MaxGhenis,  I also revised issue #3192 to reflect the correct understanding of this MD tax change.